control_pkg/tests/intregration/integration.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python
+# license removed for brevity
+import rospy
+import rostest
+import unittest
+
+from std_msgs.msg import Float64
+from sensor_msgs.msg import JointState
+from control_msgs.msg import JointControllerState
+import math
+import random
+import numpy as np
+
+
+class intTest(unittest.TestCase):
+	
+	def move_joint(self, pub, jvq):
+		for i in range (7):
+			pub.publish(jvq[i])
+	
+	def inizializer(self, j_num, string):
+		name =  '/manipulator/joint' + str(j_num) + '_position_controller/' + str(string)
+		return name
+
+	def checkpos(self, msg):
+		rospy.loginfo("Ya tut vse oki")
+		self.assertAlmostEqual(msg.set_point, msg.process_value, delta = 0.01, msg = "joint failed test")
```

Comment:
philosophically it is not right to compare these two values, but now i see that you played with tests.